### DISCUSSION TITLE
Suggestion for improving the function of pronouncing client statistics using tts

### DISCUSSION THREAD

**Discussion by artem59k** [BearWare/TeamTalk5#1644] [Ideas]
### Description

Hello. It is very good that text-to-speech now pronounces the statistics of the client. Is it possible to implement a feature in which it is necessary to request ping to the server? The function of pronouncing statistics works, but it is not always possible to catch the moment when the program automatically sends a ping. Thank you very much! Now we can easily find out the statistics of RX TX in real time using the screen reader program.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by amirmahdifard** [comment]
ping is announced sometimes randomly, but it doesn't announced moste of the times, so they should work on this more.
